tests/accessor/accessor_implicit_conversions.hpp

```diff
@@ -0,0 +1,381 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for SYCL accessor implicit conversions
+//  The main idea is to:
+//  - check if source accessor is convertible to destination accessor
+//  - and re-use generic checks for accessor constructors to verify data remains
+//    valid
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_IMPLICIT_CONVERSIONS_H
+#define SYCL_CTS_ACCESSOR_IMPLICIT_CONVERSIONS_H
+
+#include "accessor_common.h"
+
+#include <type_traits>
+
+namespace accessor_implicit_conversions {
+using namespace accessor_tests_common;
+
+namespace detail {
+template <typename SrcAccT, typename DstAccT>
```

Comment:
would you mind to add a comment about `invoke_implicit_conversion` struct?

Reply:
the comment has been added [here](https://github.com/KhronosGroup/SYCL-CTS/pull/339/commits/558b066913e80167683ea3c121e2324dd68a0c8a#diff-b2ce7dfdc46d26a407893c032262c2ba5a4790c4850ed48ef7a1e22d3ebf281bR24-R30)

Reply:
@vasilytric I've updated comments for both `invoke_implicit_conversion` and `avoid_implicit_conversion` a bit. Would you mind to review?